great_expectations/self_check/util.py

```diff
@@ -346,14 +346,12 @@
         "DECIMAL": redshifttypes.DECIMAL,
         "DOUBLE_PRECISION": redshifttypes.DOUBLE_PRECISION,
         "FOREIGN_KEY_RE": redshifttypes.FOREIGN_KEY_RE,
-        "GEOMETRY": redshifttypes.GEOMETRY,
         "INTEGER": redshifttypes.INTEGER,
         "PRIMARY_KEY_RE": redshifttypes.PRIMARY_KEY_RE,
         "REAL": redshifttypes.REAL,
         "SMALLINT": redshifttypes.SMALLINT,
         "TIMESTAMP": redshifttypes.TIMESTAMP,
         "TIMESTAMPTZ": redshifttypes.TIMESTAMPTZ,
-        "TIMETZ": redshifttypes.TIMETZ,
```

Comment:
This might just be my local env and dependency snafus, but couldn't escape `AttributeError`s here.